core/variant/variant_internal.h

```diff
@@ -418,7 +418,7 @@ class VariantInternal {
 			case Variant::PACKED_COLOR_ARRAY:
 				return get_color_array(v);
 			case Variant::OBJECT:
-				return v->_get_obj().obj;
+				return get_object(v);
```

Comment:
This change was made in the const version of `get_opaque_pointer` in PR https://github.com/godotengine/godot/pull/77410, but, unfortunately, the non-const version was missed (it was my PR, so it's my fault :-)).